pkg/cachemanager/fakecfdataclient.go

```diff
@@ -11,51 +11,50 @@ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 See the License for the specific language governing permissions and
 limitations under the License.
 */
-package fakes
+package cachemanager
```

Comment:
Why not keep this in `fakes`?

Reply:
IIRC, we wanted to keep the consumers of the `CfDataClient` interface in the same package. Is there a strong reason against having it here?

Reply:
> we wanted to keep the consumers of the CfDataClient interface in the same package.

Not quite. The statement was "interfaces are defined by the consumer", which is subtly different in that it is less prescriptive of structure. If a test object is mocking a consumer of an interface and wants to re-use its interfaces, they can live in separate packages.

https://medium.com/@cep21/preemptive-interface-anti-pattern-in-go-54c18ac0668a

Reply:
fair but this shouldn't move to `fakes` bc then there's an import cycle in our units :p : cachemanager -> fakes -> cachemanager`. Let's keep it here for now if there are no further objections.